deploy/lib/server_config.rb

```diff
@@ -949,7 +955,7 @@ def load_data(dir, options = {})
     batch = @environment != "local" && batch_override.blank? || batch_override.to_b
 
     options[:batch_commit] = batch
-    options[:permissions] = permissions(@properties['ml.app-role'], Roxy::ContentCapability::ER) unless options[:permissions]
+    options[:permissions] = permissions(@properties['ml.app-role'], Roxy::ContentCapability::ERU) unless options[:permissions]
```

Comment:
Yes, I did an upstream fetch from dev and then rebased against that. After that I began to make changes. Im dont think my other pull request for dev with this fix was accepted which may be why it was still lingering in my tree?